src/components/SessionEntry.vue

```diff
@@ -1,19 +1,111 @@
 <template>
-  <div></div>
+<!-- User message -->
+  <div v-if="!sessionActive">
+    <UserMessage/>
+  </div>
+  <!-- Entry component -->
+  <div v-else-if="sessionActive && groupData">
+    <Entry
+      :redirectTo="getRedirectTo"
+      :redirectID="getRedirectId"
+      :purpose="getPurpose"
+      :purposeParams="getPurposeParams"
+      :groupData="groupData"
+      :group="getGroup"
+      :authType="getAuthType"/>
+  </div>
 </template>
 
 <script>
+import UserMessage from "./UserMessage.vue";
+import Entry from "./Entry.vue";
+import groupAPIService from "../services/API/groupData";
 
 export default {
   name: "SessionEntry",
+  components:{UserMessage, Entry},
      props: {
-    sessionData: Object,
+    sessionData: Object
      },
-     computed: {
+     data(){
+       return {
+          sessionActive : true, // whether the timings between the session and user match
+          groupData: null, //stores details about the group
+          currentDateTime : new Date() // user's current date and time
+       }
+     },
+     computed:{
+       /** Combines session start date and start time */
+       getSessionStartDateTime(){
+         return (new Date(this.sessionData.startDate + " " + this.sessionData.startTime))
+       },
+
+       /** Combines session end date and end time */
+       getSessionEndDateTime(){
+         return (new Date(this.sessionData.startDate + " " + this.sessionData.endTime))
+       },
+
+       /** Retrieves destination platform */
+       getRedirectTo(){
+         return this.sessionData.redirectPlatform
+       },
+
+       /** Retrieves destination ID */
+       getRedirectId(){
+         return this.sessionData.redirectPlatformParams.link
+       },
+
+       /** Retrieves group name */
+       getGroup(){
+         return this.sessionData.group
+       },
```

Comment:
Either you should change the name of these. 
Because you're putting these values in computed properties, the names should not look like method names. 
Method is doing something, like "getting" something.
For example, for the group name, `getGroup` is a method. If you want to use it as a computed property, the name can simply be `groupName()`. According to the current code, how would you differentiate something like `getPurpose()` being in computed and `getSessionDateTimeInMilliseconds()` being in methods?
Both functions are doing the job of getting something.

Computed variables are not supposed to be used as method calls. They're supposed to be used as reactive variables. Let me know if it's not clear

Things like `isStartDateTimeValid()` is correct to be put in the computed properties.

Reply:
My thinking was this, computed variables are cached, so I use them for operations that are least complex (to avoid bloating). Technically, `getSessionDateTimeInMilliseconds` could also be computed if it wasn't taking in a parameter. 

But yeah, I agree about the naming. Will change that.

Reply:
My thinking was this, computed variables are cached, so I use them for operations that are least complex (to avoid bloating). Technically, `getSessionDateTimeInMilliseconds` could also be computed if it wasn't taking in a parameter. 

But yeah, I agree about the naming. Will change that.

Reply:
yes you're totally right about the segregation of logic among computed/methods
but just try to keep the naming consistent
computed properties' names should feel like variable names and not method names